### PR TITLE
Markdown view

### PR DESCRIPTION
**Pull request by eli-pinkus**
This PR does three things:

1. allows user to request the SQL for an intake sql table, `my_table` by setting the table field to `my_table@sql`
2. Creates a `Markdown` base view class for scaler values (similar to existing string view)
3. Attempts to create a subclass of `Markdown` specifically for rendering syntax highlighted sql queries (unsure how to get syntax highlighting to work though).

### CODE REVIEW

File: lumen/sources/intake_sql.py [hunk 1]
```diff
@@ -15,18 +16,26 @@ def _apply_transforms(self, source, sql_transforms):
...
         for sql_transform in sql_transforms:
             sql_expr = sql_transform.apply(sql_expr)
+            
+        print(sql_expr)
         return type(source)(**dict(source._init_args, sql_expr=sql_expr))
 
     def _get_source(self, table):
         try:
-            source = self.cat[table]
+            if table[-4:]=='@sql':
```
Comment (philippjfr) [inline lumen/sources/intake_sql.py:25]:
```suggestion
            if table.endswith('@sql'):
```

File: lumen/sources/intake_sql.py [hunk 2]
```diff
@@ -37,6 +46,8 @@ def get(self, table, **query):
         sql_transforms = query.pop('sql_transforms', [])
         source = self._get_source(table)
         source = self._apply_transforms(source, sql_transforms)
+        if table[-4:] =='@sql':
```
Comment (philippjfr) [inline lumen/sources/intake_sql.py:49]:
```suggestion
        if table.endswith('@sql'):
```

File: lumen/sources/intake_sql.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 from ..util import get_dataframe_schema
 from .base import cached
 from .intake import IntakeBaseSource, IntakeSource
+import pandas as pd
```
Comment (philippjfr) [inline lumen/sources/intake_sql.py:5]: Move import above to the external import section.

File: lumen/views/base.py [hunk 4]
```diff
@@ -604,6 +604,40 @@ def update(self, *events, invalidate_cache=True):
...
         return False
 
+class Markdown(View):
+    """
+    Renders a scaler using a Panel Markdown Pane.
+    """
+    
+    view_type = 'markdown'
+    
+#     _extension = 'markdown'
```
Comment (philippjfr) [inline lumen/views/base.py:614]: Not needed.

File: lumen/sources/intake_sql.py [hunk 5]
```diff
@@ -15,18 +16,26 @@ def _apply_transforms(self, source, sql_transforms):
         sql_expr = source._sql_expr
         for sql_transform in sql_transforms:
             sql_expr = sql_transform.apply(sql_expr)
+            
+        print(sql_expr)
```
Comment (eli-pinkus) [inline lumen/sources/intake_sql.py:20]: delete

Comment (eli-pinkus) [inline lumen/sources/intake_sql.py:20]:
```suggestion
        
```

**Pull request closed without merging**